.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+# DDCE-758
+
+##Bug fix / New feature (delete as appropriate)
+
```

Comment:
I think there's been some confusion here. You should fill out the checklist after you raise the pull request and not editing the template directly. If not, everytime anyone raises a pull request it will now have all the things you've filled in here.